src/fileEditor.cpp

```diff
@@ -62,7 +70,7 @@ FileEditor::FileEditor(const std::string& path)
 	std::ifstream file {path};
 	if (!file) {
 		endwin();
-		printf("No file specified.\n");
+		std::cout << (std::string("Error occured while trying to open " + fullFilename + ".\n").c_str());
```

Comment:
again: `std::cout << "Error occured while trying to open " << fullFilename << ".\n";`